install/installer/traf_config_setup

```diff
@@ -407,57 +412,131 @@ else
 fi
 
 #==============================================
-#Hadoop user name
-
-echo -n "Enter Hadoop admin username, default is [$ADMIN]: "
+#HADOOP_TYPE
+echo -n "Enter installed hadoop type: cloudera, hortonworks, vanilla, default is [$HADOOP_TYPE]: "
```

Comment:
suggest to ask user only Y or N , select Y if they want to install trafodion on vanilla hadoop